examples/run_FS.py

```diff
@@ -32,148 +32,163 @@
 case and 4 times in the second case.  for nScans = 2, the
 SpinCore will trigger 4 times in the first case and 8 times
 in the second case.
-'''
-#}}}
+"""
+# }}}
 from pyspecdata import *
-from numpy import *
-from . import SpinCore_pp 
-import socket
-import sys
-import time
-
-#{{{ for setting EPR magnet
-def API_sender(value):
-    IP = "jmfrancklab-bruker.syr.edu"
-    if len(sys.argv) > 1:
-        IP = sys.argv[1]
-    PORT = 6001
-    print("target IP:", IP)
-    print("target port:", PORT)
-    MESSAGE = str(value)
-    print("SETTING FIELD TO...", MESSAGE)
-    sock = socket.socket(socket.AF_INET, # Internet
-            socket.SOCK_STREAM) # TCP
-    sock.connect((IP, PORT))
-    sock.send(MESSAGE)
-    sock.close()
-    print("FIELD SET TO...", MESSAGE)
-    time.sleep(3)
-    return
-#}}}
+from pylab import *
+import os
+import SpinCore_pp
+from SpinCore_pp.ppg import run_spin_echo
+from datetime import datetime
+from Instruments.XEPR_eth import xepr
+import numpy as np
+import h5py
 
-field_axis = linspace(3490.0,3510.0,5,endpoint=False)
+field_axis = r_[3490.0:3510.0:1.0]
```

Comment:
should we generate this using ppt and the nu_ESR and gamma_eff?:
`left = (config_dict['ppt']*config_dict['nu_ESR']*(config_dict['gamma_eff']^-1)) - 3
 right = (config_dict['ppt']*config_dict['nu_ESR']*(config_dict['gamma_eff']^-1)) + 3
 field_axis = r_[left: right: 1.0]`